juniper_official/Solutions/EVPN-VXLAN/check-flabels-count-netconf.rule

```diff
@@ -0,0 +1,103 @@
+/*
+ * Collects flabels token count periodically and notifies anomalies when flabels token 
+ * count is above static threshold.
+ *
+ * One input control detection
+ * 
+ *   1) token-threshold, is the threshold that causes the rule to report an anomaly. By 
+ *      default it's 94000. This rule will set a dashboard color to red when flabels token
+ *      count is greater than token-threshold. Otherwise color is set to green.
+ */
```

Comment:
s/One input control detection/One input controls detection/g
s/token-threshold,/token-threshold variable,/g

Reply:
updated the rule